### ISSUE TITLE
[Experimental] Classes and Features Munch missing items

### ISSUE THREAD

**Issue by MaxPat931** [MrPrimate/ddb-importer#147]
Importing Class Features via the Munch is missing items that would be imported via character import

Extra Attack
Bonus Cantrip (Light Domain)
Bonus Cantrip (Circle of the Land)
Bonus Cantrips (The Celestial)
Bonus Proficiencies (Order Domain)
Bonus Proficiencies (Tempest Domain)
Bonus Proficiencies (Twilight Domain)
Bonus Proficiencies (War Domain)
Bonus Proficiencies (Way of the Drunken Master)
Bonus Proficiencies (Assassin)
Bonus Proficiencies (College of Lore)
Bonus Proficiencies (College of Swords)
Bonus Proficiencies (College of Valor)
Bonus Proficiencies (Rune Knight)
Bonus Proficiency (Samurai)
Bonus Proficiency (Cavalier)
Bonus Proficiency (Nature Domain)
Bonus Proficiency (Life Domain)
Bonus Proficiencies (Forge Domain)
Bonus Proficiency (Death Domain)
Tool Proficiency: Smith's Tools
Tool Proficiency: Woodcarver's Tools
Tool Proficiency: Brewer's Supplies
Channel Divinity (Paladin)
Channel Divinity: Abjure Enemy
Channel Divinity: Abjure the Extraplanar
Channel Divinity: Champion Challenge
Channel Divinity: Conquering Presence
Channel Divinity: Control Undead
Channel Divinity: Dreadful Aspect
Channel Divinity: Emissary of Peace
Channel Divinity: Fury of the Tides
Channel Divinity: Inspiring Smite
Channel Divinity: Marine Layer
Channel Divinity: Nature's Wrath
Channel Divinity: Peerless Athlete
Channel Divinity: Rebuke the Violent
Channel Divinity: Sacred Weapon
Channel Divinity: Turn the Faithless
Channel Divinity: Turn the Tide
Channel Divinity: Turn the Unholy
Channel Divinity: Turn Undead
Channel Divinity: Watcher’s Will
Channel Divinity: Vow of Enmity
Domain Spells (Arcana Domain)
Domain Spells (Death Domain)
Domain Spells (Forge Domain)
Domain Spells (Grave Domain)
Domain Spells (Knowledge Domain)
Domain Spells (Life Domain)
Domain Spells (Light Domain)
Domain Spells (Nature Domain)
Domain Spells (Order Domain)
Domain Spells (Peace Domain)
Domain Spells (Tempest Domain)
Domain Spells (Trickery Domain)
Domain Spells (Twilight Domain)
Domain Spells (War Domain)
Primal Companion

Also missing items in the following categories that are choices by PC:
Ki Options				Monk
Invocations				Warlock
Maneuvers				Battle Master
Infusions					Artificer
Arcane Shot Options		Arcane Archer
Totem Spirits				Path of the Totem Warrior
Trick Shots				Gunslinger
Psionic Powers (Psi Warrior)	Psi Warrior
Psionic Powers (Soulknife)	Soulknife
Runes					Rune Knight
Fighting Style				Fighter, Ranger, Paladin
Metamagic Options		Sorcerer
Pact Boons				Warlock
Blood Curses				Blood Hunter
Primal Rites				Blood Hunter
Mutagens				Order of the Mutant

There are also several items imported that are listed as subclass options, but are class options (some examples):
Fighting Style (Echo Knight)
Spellcasting (Armorer)
Timeless Body (Circle of Stars)
Unarmored Defense (Path of the Berserker)

**Comment by MrPrimate** [comment]
Hi @MaxPat931 this is a current limitation of the way this is implemented currently and requires a large re-write to improve.

**Comment by MrPrimate** [comment]
I will also break this into 4 issues, as they are different bugs/enhancements to solve.

**Comment by MaxPat931** [comment]
Yep, I figured that would be the case. I understand this feature isn't a priority right now, I just wanted to make note of my findings while I was working on it this past week.

**Comment by MrPrimate** [comment]
It's a very useful list, thanks!

**Comment by MaxPat931** [comment]
Bonus Proficiencies have been included, some Domain Spells have been included, I've updated the initial comment to reflect that.
Also removed the feature options (ki, invocations, etc) to create a new ticket
